jobs/JREGIONAL_AQM_LBCS

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+#
+#-----------------------------------------------------------------------
+#
+# This script generates both the initial conditions (ICs) and the lateral
+# boundary conditions (LBCs) files for the air quality model (AQM) from
+# files stored either in the system directory or on mass storage (HPSS).
+#
+#-----------------------------------------------------------------------
+#
+
+#
+#-----------------------------------------------------------------------
+#
+# Source the variable definitions file and the bash utility functions.
+#
+#-----------------------------------------------------------------------
+#
+. $USHdir/source_util_funcs.sh
+source_config_for_task "task_get_extrn_lbcs|task_make_orog|task_make_lbcs|cpl_aqm_parm" ${GLOBAL_VAR_DEFNS_FP}
```

Comment:
Thanks for sticking to this format for sourcing variables! I hope it was not much of an inconvenience when a task refers to many sections of the config file.